project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module

```diff
@@ -475,7 +475,7 @@ function _c4m_document_widget_document_file($file_id, $is_quick_post = FALSE) {
   // PURL options.
   $options_context = array(
     'purl' => array(
-      'disabled' => FALSE,
+      'enabled' => TRUE,
```

Comment:
@ybaras 
I've tested this, and verified purl is properly created when we have `'enabled' => TRUE`.
Please approve and update your PR.